src/parser/core/modules/NotCasting.tsx

```diff
@@ -0,0 +1,184 @@
+import {t} from '@lingui/macro'
+import {Trans} from '@lingui/react'
+import {Action} from 'data/ACTIONS'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import Downtime from 'parser/core/modules/Downtime'
+import {GlobalCooldown} from 'parser/core/modules/GlobalCooldown'
+import {Timeline} from 'parser/core/modules/Timeline'
+import React from 'react'
+import {Table, Button} from 'semantic-ui-react'
+import {Data} from './Data'
+
+//value to be added to the gcd to avoid false positives. 100ms for caster tax, 50ms for gcd jitter.
+const GCD_ERROR_OFFSET = 150
+const BASE_RECAST = 2500
+
+interface Window {
+	start: number,
+	stop?: number,
+	recastTime: number,
+}
+
+export class NotCasting extends Analyser {
+	static override handle = 'notcasting'
+	static override title = t('core.notcasting.title')`Times you did literally nothing`
+
```

Comment:
While this was a BLM-only thing, I wasn't as worried about this title, but if we're proposing making it general to all casting jobs, we really need a better way to label this...